CHANGES.md

```diff
@@ -7,6 +7,7 @@ number or not.
 ## Gameplay and UI
 
 - Default teams now have logos
+- International players from 85 countries, and better names for American players
```

Comment:
Done. You should see a new PR now. Shall I close this one? Or close both at the same time when the other one is finished?